modules/ROOT/pages/manage-cli.adoc

```diff
@@ -0,0 +1,567 @@
+= Connect and manage the {product}
+:navtitle: Connect and manage the CLI
+:page-aliases: ROOT:managing.adoc, ROOT:astra-streaming-cli.adoc, ROOT:administration.adoc
+
+////
+Author's note: The following pages used to serve as de facto command references:
+
+ROOT:managing.adoc, ROOT:astra-streaming-cli.adoc, ROOT:administration.adoc
+
+However, since the new command reference pages are programmatically generated, there isn't a good place to redirect them to (yet).
+Once we have quickstarts for Astra DB, Astra Streaming, and Astra administration, we can move the page aliases.
+////
+
+Learn how to connect and run commands with the {product}.
+
+If you haven't installed the {product} yet, see xref:ROOT:install.adoc[].
+
+== Get an application token
+
+The {product} requires an xref:astra-db-serverless:administration:manage-application-tokens.adoc[application token] to connect to {product-short}.
+
+xref:astra-db-serverless:administration:manage-application-tokens.adoc#org-token[Generate an application token] and copy the token string (`AstraCS:…`) before moving on to the next section.
+
+[TIP]
+====
+If this is your first time using the {product}, {company} recommends that you connect to a non-production organization first.
+That way, you can assign your application token a more expansive role, such as *Organization Administrator*, so that you can try out all of the {product}'s features without putting production data at risk.
+
+If you're using the {product} in production, {company} strongly recommends that you assign your application token only the bare minimum permissions required to perform the intended tasks.
```

Comment:
```suggestion
If you're using the {product} in production, {company} strongly recommends that you follow the principle of least privilege: Make sure your application token's role provides only the minimum required permissions.
For example, don't grant an administrator-level role to a token that won't be used to execute those types of commands.
```

Nitpick cause you can't assign individual permissions to a token, it has to be a role with a set of permissions.